publish/binaries/BUILD.bazel

```diff
@@ -31,6 +31,7 @@ ALL_BINARIES = {
     "orchestrator": "//rs/orchestrator",
     "sandbox_launcher": "//rs/canister_sandbox:sandbox_launcher",
     "sns": "//rs/sns/cli:sns",
+    "sync-with-released-nervous-system-wasms": "//rs/nervous_system/tools/sync-with-released-nervous-system-wasms",
```

Comment:
Why does this tool need to be published? Wouldn't we just use this internally in the ic repo where we can just `bazel run //rs/nervous_system/tools/sync-with-released-nervous-system-wasms ...` it?

Reply:
We want this action to be done via a bot that would then create a PR in this repo. 

I was thinking that the bot could be added to https://github.com/dfinity-ops/release

Is there a better way?

Reply:
Why not via a GitHub action in this repo similar to what @NikolaMilosa is doing for `mainnet_revisions.json` in https://github.com/dfinity/ic/pull/3085?

Reply:
Agreed. But adding it to that repo would require publishing the tool from this repo, I think. 

So, may I get an approval, @basvandijk?

Reply:
WDYM with "that repo". What Nikola is doing is adding a GH action to "this repo".

Reply:
Ah, so I misunderstood. 

Yes, keeping all this within the same repo seems better. 

(I wonder how we can build / run a bazel target from an action. But let's move that discussion outside this PR.)

So I'm removing the publishing of this target from this PR.

Thanks!

Reply:
```suggestion
```

Reply:
>I wonder how we can build / run a bazel target from an action. But let's move that discussion outside this PR.

```
bazel run --config=ci //rs/nervous_system/tools/sync-with-released-nervous-system-wasms ...
```